### PR TITLE
Documentation fixes before 2.1 release

### PR DESCRIPTION
**Pull request by Didainius**

### CODE REVIEW

**Comment by lvirbalas** [comment]
Thanks, this will help the newcomers, who just want to clone the repo without prior SSH key setup.

Just piggy backing on this commit, could you also improve this doc page:

http://localhost:4567/docs/providers/vcd/r/vapp_vm.html#vapp_network_name
> ip - (Optional) The IP to assign to this vApp. Must be an IP address or one of dhcp, allocated or none. If given the address must be within the static_ip_pool set for the network. If left blank, and the network has dhcp_pool set with at least one available IP then this will be set with DHCP.

I suggest quoting the IP types, because it's not totally clear that these are keywords. Suggestion:

> Must be an IP address or one of "dhcp", "allocated" or "none".

p.s. The URL below is in the Docker webserver that is created with `make website`.

**Comment by lvirbalas** [comment]
One more thing, @vbauzysvmware README.md says:
> * resource/vcd_vapp_vm add additional validation to check if vApp template is OK [GH-157] 

But #157 is this:
https://github.com/terraform-providers/terraform-provider-vcd/issues/157

**Comment by vbauzysvmware** [comment]
@lvirbalas and what is issue with #157 ?

**Comment by lvirbalas** [comment]
It is not clear that this link is correct, because the issue name is different and it is still open. I'd suggest relinking to the PR as per guidelines. The PR is clearer and references issue in the first line for those who want more history.

**Comment by vbauzysvmware** [comment]
@Didainius Could update to 157 to #151 Add check for vApp template to be valid

**Comment by Didainius** [comment]
> I suggest quoting the IP types, because it's not totally clear that these are keywords.

Agreed. I tried to emphasize using back ticks only, but it was still purely visible so left those quoted.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ IMPROVEMENTS:
 
 BUG FIXES:
 * resource/vcd_vapp, resource/vcd_vapp_vm add vApp status handling when environment is very fast [GH-68]
-* resource/vcd_vapp_vm add additional validation to check if vApp template is OK [GH-157] 
+* resource/vcd_vapp_vm add additional validation to check if vApp template is OK [GH-151]
```
Comment (lvirbalas) [inline CHANGELOG.md]: Oops! 151 is a PR in go-vcloud-director project, so the above turned out linking to a wrong *issue* in terraform-provider-vcd

Comment (Didainius) [reply CHANGELOG.md]: Done. force pushed without the changes at all.

**Comment by lvirbalas** [comment]
@Didainius let's add a comment to CHANGELOG.md similar to the one we had in v2.0.0:

`Please look for "v2.1+" keyword in documentation which is used to emphasize new features.`

**Pull request merged**